M2/Macaulay2/m2/document.m2

```diff
@@ -244,21 +245,22 @@ fSeq := new HashTable from {
     (2, symbol ~       ) => s -> (toString s#1, " ", toString s#0), -- postfix operator
     (2, symbol !       ) => s -> (toString s#1, " ", toString s#0), -- postfix operator
 
-    -- ScriptedFunctors
-    (4, class, ScriptedFunctor, ZZ) => s -> (
+    -- Functors
+    -- TODO: make sure hh.argument is also covered
+    (4, class, Functor, ZZ) => s -> (
 	hh := s#0;
 	if hh.?subscript
 	then (toString hh, "_", toString s#1, "(", toString s#2, ",", toString s#3, ")")
 	else (toString hh, "^", toString s#1, "(", toString s#2, ",", toString s#3, ")")),
-    (3, class, ScriptedFunctor, ZZ) => s -> (
+    (3, class, Functor, ZZ) => s -> (
 	hh := s#0;
 	if hh.?subscript
```

Comment:
Isn't it just scripted functors that take subscripts, and not functors?  So this code is strange.

Reply:
This is a good point. I think just renaming ScriptedFunctor to Functor would be simpler and also resolve this.

Reply:
I fixed this for now.

Reply:
I think until now the entire point of `ScriptedFunctor` has been a technical thing, namely storing different functions for subscript and superscript, but there are many "functions" in Macaulay2 that are really functors, as in they take objects to objects and morphisms to morphisms, like `basis` or `truncate`. I think making that more explicit is good. This also helps with the general goal of keeping Macaulay2 syntax close to the mathematical notation.